pipeline_dp/dp_computations.py

```diff
@@ -529,6 +536,44 @@ def describe(self) -> str:
                 f"l2_sensitivity={self.sensitivity}")
 
 
+class MeanMechanism:
+    """Computes DP mean.
+
+    It computes DP mean as a ratio of DP sum and DP count. For improving
+    utility the normalization to [min_value, max_value] is performed.
+    It works in the following way:
+    1. normalized_sum = \sum(x_i-mid), where mid = (min_value+max_value)/2.
```

Comment:
Does Python support Latex?

Reply:
Python comments are not treated as Latex :(. I've removed "\\".